NEXT_CHANGELOG.md

```diff
@@ -115,6 +115,13 @@ By [@Geal](https://github.com/geal) in https://github.com/apollographql/router/p
 
 ## 🐛 Fixes
 
+### Don't send header names on Studio if `send_headers` is `none` ([Issue #2403](https://github.com/apollographql/router/issues/2403))
+
+Before when `send_headers` was set to `none` we sent header names with empty header values. Now we don't send anything to Studio.
+
```

Comment:
Suggest adding minimal example yaml config.

Reply:
Fixed in https://github.com/apollographql/router/pull/2425/commits/bb88f4b716998620d75109ff75731e25b9859709